catalogs/catalog-lakehouse-paimon/src/test/java/com/datastrato/gravitino/catalog/lakehouse/paimon/integration/test/CatalogPaimonBaseIT.java

```diff
@@ -57,7 +72,75 @@ private void resetSchema() {
 
   protected abstract Map<String, String> initPaimonCatalogProperties();
 
-  private void clearTableAndSchema() {}
+  @Test
+  void testPaimonSchemaOperations() throws DatabaseNotExistException {
+    SupportsSchemas schemas = catalog.asSchemas();
+
+    // list schema check.
+    Set<String> schemaNames = new HashSet<>(Arrays.asList(schemas.listSchemas()));
```

Comment:
please don't rely on `schemaName`, it's out of the test

Reply:
it's odd to test list schema first

Reply:
updated.